packages/itwin/map-layers/src/ui/widget/SubLayersTree.tsx

```diff
@@ -114,7 +111,7 @@ export function SubLayersTree(props: SubLayersTreeProps) {
   // it selects/deselects node when checkbox is checked/unchecked and vice versa.
   // `useDisposable` takes care of disposing old event handler when new is created in case 'nodeLoader' has changed
   // `React.useCallback` is used to avoid creating new callback that creates handler on each render
-  const eventHandler = useDisposable(
+  const eventHandler = useOptionalDisposable(
```

Comment:
was this due to deprecation?

Reply:
yes, then useOptionalDisposable was deprecated in React 18, so I refactored to just a `useEffect`